ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -332,6 +373,52 @@ class JITModBuilder {
     }
   }
 
+  // Given an `IRBlock`, find and return the LLVM data structures for the basic
+  // block and its parent function.
+  std::pair<Function *, BasicBlock *> getLLVMAOTFuncAndBlock(IRBlock *IB) {
+    Function *F = AOTMod->getFunction(IB->FuncName);
+    if (!F)
```

Comment:
This looks more like an `assert` in disguise to me?

Reply:
Fixed in 1a67a96.